core/scheduler/service/job_run_service.go

```diff
@@ -348,6 +348,9 @@ func (s *JobRunService) raiseJobRunStateChangeEvent(jobRun *scheduler.JobRun) {
 		schedulerEvent, err = event.NewJobRunSuccessEvent(jobRun)
 	case scheduler.StateFailed:
 		schedulerEvent, err = event.NewJobRunFailedEvent(jobRun)
+	default:
+		s.l.Error("unrecognised job run state, to raise job run state change event")
```

Comment:
let's also print what is this unrecognized state

Reply:
suggestion on the message:
```go
s.l.Error("state [%s] is unrecognized, event is not published", jobRun.State)
```

Reply:
cool